<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>803</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QSplitter" name="mainSplitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>1</horstretch>
        <verstretch>3</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QFrame" name="leftFrame">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>1</horstretch>
         <verstretch>3</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QSplitter" name="leftSplitter">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QFrame" name="topLeft">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QPushButton" name="importBtn">
              <property name="text">
               <string>Import Files +</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTreeWidget" name="fileList">
              <property name="headerHidden">
               <bool>false</bool>
              </property>
              <property name="columnCount">
               <number>2</number>
              </property>
              <column>
               <property name="text">
                <string>File</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Status</string>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QFrame" name="bottomLeft">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QToolBox" name="infoToolBox">
              <property name="currentIndex">
               <number>2</number>
              </property>
              <widget class="QWidget" name="logPage">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>0</y>
                 <width>350</width>
                 <height>102</height>
                </rect>
               </property>
               <attribute name="label">
                <string>Log</string>
               </attribute>
               <widget class="QTextEdit" name="logView">
                <property name="geometry">
                 <rect>
                  <x>0</x>
                  <y>0</y>
                  <width>500</width>
                  <height>150</height>
                 </rect>
                </property>
                <property name="lineWrapMode">
                 <enum>QTextEdit::NoWrap</enum>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </widget>
              <widget class="QWidget" name="instrPage">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>0</y>
                 <width>350</width>
                 <height>102</height>
                </rect>
               </property>
               <attribute name="label">
                <string>Instructions</string>
               </attribute>
              </widget>
              <widget class="QWidget" name="faqPage">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>0</y>
                 <width>350</width>
                 <height>102</height>
                </rect>
               </property>
               <attribute name="label">
                <string>FAQs</string>
               </attribute>
               <widget class="QLabel" name="faqText">
                <property name="geometry">
                 <rect>
                  <x>6</x>
                  <y>3</y>
                  <width>341</width>
                  <height>101</height>
                 </rect>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QFrame" name="rightFrame">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>1</horstretch>
         <verstretch>3</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QSplitter" name="rightSplitter">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QFrame" name="topRight">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QListWidget" name="faceList">
              <property name="iconSize">
               <size>
                <width>96</width>
                <height>96</height>
               </size>
              </property>
              <property name="resizeMode">
               <enum>QListView::Adjust</enum>
              </property>
              <property name="viewMode">
               <enum>QListView::IconMode</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="continueBtn">
              <property name="text">
               <string>Continue to Annotate Face Chains &gt;&gt;</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QFrame" name="bottomRight">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QLabel" name="timeLabel">
              <property name="text">
               <string>00:00:00/00:00:00</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="horizontalSlider">
              <property name="maximum">
               <number>100</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>803</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Main_Page">
    <property name="title">
     <string>Home</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuInstrcutions">
    <property name="title">
     <string>Instrcutions</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
   </widget>
   <addaction name="menu_Main_Page"/>
   <addaction name="menuInstrcutions"/>
   <addaction name="menuOptions"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
